kafka-ssl/README.md

```diff
@@ -0,0 +1,133 @@
+# Debezium - Kafka with SSL example
+
+This demo automatically deploys the topology of services as defined in the [Debezium Tutorial](https://debezium.io/documentation/reference/stable/tutorial.html) with Kafka SSL enabled.
+
+## Configuration
+
+The kafka instance will be deployed using the certificates at the './resources' folder:
+```yaml
+  kafka:
+    image: quay.io/debezium/kafka:${DEBEZIUM_VERSION}
+    ports:
+     - "9092:9092"
+    links:
+     - zookeeper
+    environment:
+     - ZOOKEEPER_CONNECT=zookeeper:2181
+     - KAFKA_LISTENER_SECURITY_PROTOCOL_MAP=BROKER:PLAINTEXT,SSL:SSL,CONTROLLER:PLAINTEXT
+     - KAFKA_LISTENERS=BROKER://0.0.0.0:9093,SSL://0.0.0.0:9092,CONTROLLER://0.0.0.0:9094
+     - KAFKA_ADVERTISED_LISTENERS=SSL://localhost:9092,BROKER://localhost:9093
+     - KAFKA_SSL_CLIENT_AUTH=required
+     - KAFKA_SSL_KEYSTORE_LOCATION=/opt/config/ssl/kafka-ssl-keystore.p12
+     - KAFKA_SSL_KEYSTORE_PASSWORD=top-secret
+     - KAFKA_SSL_KEYSTORE_TYPE=PKCS12
+     - KAFKA_SSL_TRUSTSTORE_LOCATION=/opt/config/ssl/kafka-ssl-truststore.p12
+     - KAFKA_SSL_TRUSTSTORE_PASSWORD=top-secret
+     - KAFKA_INTER_BROKER_LISTENER_NAME=BROKER
+    volumes:
+      - ./resources:/opt/config/ssl:z
+```
+
+The certificates will be mounted at the folder "/opt/config/ssl".
+
+Then, when configuring the debezium connector, we need to also mount these certificates and properly configure the connector:
+
+```yaml
+  connect:
+    image: quay.io/debezium/connect:${DEBEZIUM_VERSION}
+    ports:
+      - "8083:8083"
+    environment:
```

Comment:
IMHO you also need producer and consumer prefixes, see https://docs.confluent.io/platform/current/security/protect-data/encrypt-tls.html#encryption-ssl-connect